lib/gooddata/bricks/pipeline.rb

```diff
@@ -87,6 +88,15 @@ def self.rollout_brick_pipeline
         RolloutBrick
         ])
       end
+
+      def self.hello_world_brick_pipeline
+        prepare([
+          DecodeParamsMiddleware,
```

Comment:
warning: Style/IndentArray: Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis. (rubocop)